tools/Google.Cloud.Tools.ReleaseManager/CreateClientsCommand.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using Google.Cloud.Tools.Common;
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace Google.Cloud.Tools.ReleaseManager;
+
+public class CreateClientsCommand : CommandBase
+{
+    private const string PublishTargetFramework = "netstandard2.1";
+
+    public CreateClientsCommand()
+        : base("create-clients", "Checks that we can create clients for a pure-GAPIC package", "id")
+    {
+    }
+
+    protected override void ExecuteImpl(string[] args)
+    {
+        string id = args[0];
+        var catalog = ApiCatalog.Load();
+        if (!catalog.TryGetApi(id, out var api))
+        {
+            throw new UserErrorException($"No such API: {id}");
+        }
+        if (api.Generator != GeneratorType.Micro)
+        {
+            Console.WriteLine($"{id} is not a GAPIC-generated API; skipping client creation test.");
+        }
+
+        var assembly = PublishAndLoadAssembly(id);
+        if (assembly is null)
```

Comment:
nit: Maybe a note on when this could happen?

Reply:
Ooh, this was left over from a previous iteration. Will remove.